flight/waypoint/goto.py

```diff
@@ -53,16 +49,13 @@ async def move_to(
             drone_long: float = position.longitude_deg
             drone_alt: float = position.relative_altitude_m
 
+            total_distance: float = calculate_distance(
+                drone_lat, drone_long, drone_alt, latitude, longitude, altitude
+            )
             #  accurately checks if location is reached and stops for 15 secs and then moves on.
-            if (
-                (round(drone_lat, int(6 * fast_param)) == round(latitude, int(6 * fast_param)))
-                and (
-                    round(drone_long, int(6 * fast_param)) == round(longitude, int(6 * fast_param))
-                )
-                and (round(drone_alt, 1) == round(altitude, 1))
-            ):
+            if total_distance < 5:
```

Comment:
In the SUAS rules, we have a tolerance of 25ft for each waypoint. Is there a reason you chose 5 meters? Also make this value a constant

Reply:
5 meters = 16.4 feet, just wanted to add a little tolerance instead of making it 25ft. I'll change it to 20ft/6m